ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.tsx

```diff
@@ -126,11 +125,7 @@ function NetworkGraphPage() {
 
     // We will update the poll epoch after 30 seconds to update the node count for a cluster
     useInterval(() => {
-        setPollEpoch(pollEpoch + 1);
```

Comment:
Since the only use of the `pollEpoch` variable was to trigger an effect, we can instead simplify this by moving the update callback directly into the `useInterval`.